Lib/uuid.py

```diff
@@ -52,6 +52,9 @@
 
 __author__ = 'Ka-Ping Yee <ping@zesty.ca>'
 
+_notAIX = not sys.platform.startswith("aix")
```

Comment:
Please rename the constant to _AIX and use "not _AIX".

Reply:
done.